amlip_docs/rst/user_manual/nodes/agent.rst

```diff
@@ -91,12 +116,35 @@ Steps
             // Create Server Node
             eprosima::amlip::node::agent::ServerNode Client_node(
                 "CppServerNode_Manual",
-                { listening_address });
+                { listening_address },
+                100);
```

Comment:
If set explicitly, I wouldn't use the same domain for client and server, in case someone is testing in the same machine/LAN.